lib/services/messenger/translators/discourse.ts

```diff
@@ -443,6 +443,16 @@ export class DiscourseTranslator extends TranslatorScaffold implements Translato
 						details.post.raw, MetadataEncoding.HiddenMD, this.metadataConfig
 					);
 			}
+			const quoteFinder = /\[quote="([^,"]*).*"]\s*([\s\S]*)\[\/quote]/;
```

Comment:
Check https://app.frontapp.com/open/cnv_rxwalv for an example, I paste the body of a message as it appeared in Front below:

```
[quote="lucianbuzzo, post:14, topic:3101"]

curl https://api.resin.io/ping

[/quote]

Hi @lucianbuzzo

here is the output from the device :

[code]

root@a7b6c21:~# curl https://api.resin.io/ping

OK

[/code]
```

A few questions:

- Using `>` quoted phrases in Front should render correctly in Discourse, without needing a translation back to the `[quote]` tag format, correct? Since Discourse support markdown
- Do we handle `[code]` Discourse -> Front translations in the recently merged PR that handles code snippets?

Reply:
Checked the `>`, it's fine.
Added code for the [code] format